java/src/jmri/NamedBean.java

```diff
@@ -76,18 +81,18 @@
      * @param listenerRef - A textual reference for the listener, that can be
      *                    presented to the user when a delete is called
      */
-    public void addPropertyChangeListener(java.beans.PropertyChangeListener l, String name, String listenerRef);
+    public void addPropertyChangeListener(@Nonnull java.beans.PropertyChangeListener l, String name, String listenerRef);
```

Comment:
java.beans.PropertyChangeSupport.addPropertyChangeListener(...) is explicitly documented as accepting null for the listener.